hrui/CRM/Core/BAO/Navigation.php

```diff
@@ -567,33 +567,31 @@ public static function createNavigation($contactID) {
       // get home menu from db
       $homeParams = array('name' => 'Home');
       $homeNav = array();
-      $homeIcon = '<span class="chr_logo chr_logo--default-color"></span>';
+      $homeIcon = '<span class="crm-logo-sm" ></span>';
       self::retrieve($homeParams, $homeNav);
       if ($homeNav) {
-        if (array_key_exists('query', parse_url($homeNav['url']))) {
-          list($path, $q) = explode('?', $homeNav['url']);
-        } else {
-          $q = NULL;
-          $path = $homeNav['url'];
-        }
-
+        list($path, $q) = explode('?', $homeNav['url']);
```

Comment:
Are you sure we should remove this override? It was added as part of https://github.com/civicrm/civihr/pull/924 to fix a Notice due to the fact that the T&A URLs don't have a query string. Does that still hold true?

Reply:
good catch, will put back